.readthedocs.yaml

```diff
@@ -0,0 +1,41 @@
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+  jobs:
+    post_checkout:
+      # Cancel building pull requests when there aren't changed in the docs directory or YAML file.
+      # You can add any other files or directories that you'd like here as well,
+      # like your docs requirements file, or other files that will change your docs build.
```

Comment:
like your docs requirements file, or other files that will change your docs build.

->

such as docs requirements file or other files that affect the docs build.